VideoAPI/VideoApi.DAL/Commands/UpdateMeetingRoomCommand.cs

```diff
@@ -9,20 +9,22 @@ namespace VideoApi.DAL.Commands
     public class UpdateMeetingRoomCommand : ICommand
     {
         public UpdateMeetingRoomCommand(Guid conferenceId, string adminUri, string judgeUri, string participantUri,
-            string pexipNode)
+            string pexipNode, string telephoneConferenceId)
         {
             ConferenceId = conferenceId;
             AdminUri = adminUri;
             JudgeUri = judgeUri;
             ParticipantUri = participantUri;
             PexipNode = pexipNode;
+            TelephoneConferenceId = telephoneConferenceId;
         }
 
-        public Guid ConferenceId { get; set; }
-        public string AdminUri { get; set; }
-        public string JudgeUri { get; set; }
-        public string ParticipantUri { get; set; }
-        public string PexipNode { get; set; }
+        public Guid ConferenceId { get; }
+        public string AdminUri { get; }
+        public string JudgeUri { get; }
+        public string ParticipantUri { get; }
+        public string PexipNode { get; }
+        public string TelephoneConferenceId { get; }
```

Comment:
is there a reason to make it as string?

Reply:
Kinly return an 8 digit numeric value as a string

`"pstnPin":"62695113"`